src/pages/GE_TokenizeCard.page

```diff
@@ -18,7 +18,7 @@
         let domain = arr[0].replace('--npsp', '');
         domain = domain.replace('--c', '');
         const lightningOrigin = `https://${domain}.lightning.force.com`;
-        let nameOnCard = 'test name';
+        let nameOnCard = 'Not provided';
```

Comment:
Since the VF page is changed in this PR, used the opportunity to change the default value from `test name` to `Not provided`. I can revert back the constant if that is required.

Reply:
I like making the switch here, but I think the value needs to match what Elevate is expecting, which I believe is "[Not Provided]". I'm not actually clear on if the string we use needs to match exactly some expected value, but to be safe can we use that string (with the square brackets) until we hear otherwise?

Reply:
The new change with the brackets included looks good!  Thanks Radmila!